sirepo/pkcli/test_http.py

```diff
@@ -378,18 +390,20 @@ def create_task(self):
         async def _run():
             _sims.append(self)
             # Must be set here once we are in the _run() task
-            self._task_id = str(id(asyncio.current_task()))[-4:]
+            self._task_id = _task_id(asyncio.current_task())
```

Comment:
`_current_task_id` would be a better encapsulation.

Reply:
But [it isn't always the current task](https://github.com/radiasoft/sirepo/pull/3557/files#diff-0b269c6d8875a46dc3afb0eb98ebc99343ccda7ad3346e971bd936e842d305feR207)